Makefile

```diff
@@ -81,7 +81,10 @@ bench:
 
 .PHONY: system-tests
 system-tests: $(PYTHON_BIN) apm-server.test
-	INTEGRATION_TESTS=1 TZ=UTC $(PYTHON_BIN)/nosetests $(NOSETESTS_OPTIONS) $(SYSTEM_TEST_TARGET)
+    # we need to disable libbeat's tracer so it doesn't try to fetch config when it shouldn't
+    # longterm solution is to adapt libbeat so we can inject apm-server's tracer
+    # this won't affect apm-server tests, as the `initTracer` function will override ELASTIC_APM_ACTIVE
+	ELASTIC_APM_ACTIVE=false INTEGRATION_TESTS=1 TZ=UTC $(PYTHON_BIN)/nosetests $(NOSETESTS_OPTIONS) $(SYSTEM_TEST_TARGET)
```

Comment:
With this update the libbeat tracer would always be enabled, and only disabled in the tests. To avoid changing tracing behavior, you could set `os.Setenv("ELASTIC_APM_ACTIVE", "false")` in the `cmd/root.go`, disabling the libbeat tracer until the behavior is fixed.